docs/translating.md

```diff
@@ -58,6 +58,14 @@ You can instead use:
 {% blocktrans %}This string will have {{ value }} inside.{% endblocktrans %}
 ```
 
+## Frontend Translations
+The first step in translating the frontend is to create the translation
+files using `./tools/make-frontend-messages`. This command will create
+translation files under `static/locale`, the location can be changed by
+passing an argument to the command, however make sure that the location is
+publically accessible since these files are loaded through XHR in the
+frontend which will only work with publically accessible resources.
+
```

Comment:
Can you detail more of the overall workflow for using this here?  It looks like `./tools/make-frontend-messages` extracts the strings from the templates.  How does one upload that data to transifex for translation (and how do we combine it with the Django template data)?  How does one download the data from Transifex?  Is there something we need to do to merge the messages files from this and the Django one?

It might makes sense in answering these questions to write scripts to do any steps of the workflow that are currently cumbersome.